packages/cdk-route53-record/lib/index.ts

```diff
@@ -0,0 +1,54 @@
+import * as cdk from "@aws-cdk/core";
+import * as route53 from "@aws-cdk/aws-route53";
+import * as targets from "@aws-cdk/aws-route53-targets";
+import * as elbv2 from "@aws-cdk/aws-elasticloadbalancingv2";
+
+export interface CdkRoute53RecordProps {
+  targetType: string;
+  loadBalancerArn: string;
+  hostedZoneDomainNames: string[];
+  dnsRecords: string[];
+}
+
+export class CdkRoute53Record extends cdk.Construct {
```

Comment:
This class name sounds a little generic to me since it is specifically for load balancer target. Or were you planning on building this out to include every possible target?

Reply:
The latter.